terminus-op/Chart.yaml

```diff
@@ -0,0 +1,9 @@
+apiVersion: v2
+name: terminus-op
```

Comment:
If it is not a operator shouldn't we just call it terminusdb? And if you intend to make an operator, what do you want to make the CRD do exactly? I'm not against it by definition but would like to know the use of it.

Reply:
I am still actively reading about Terminus Architecture, The CRD will ideally have clusters of terminus DB nodes that'd be able to act as standalone instances. Additionally, If terminus DB has some form of [gossip](https://www.google.com/url?sa=t&rct=j&q=&esrc=s&source=web&cd=&cad=rja&uact=8&ved=2ahUKEwj58IuQ6dv2AhXcSWwGHd_yCfIQFnoECA4QAQ&url=https%3A%2F%2Fdocs.datastax.com%2Fen%2Fcassandra-oss%2F3.x%2Fcassandra%2Farchitecture%2FarchGossipAbout.html&usg=AOvVaw3pNc4Jq5kDw7drpLfAUrOq) they should be able to do so if they are in the same cluster.